definitions/pipelines/panel_of_normals.cwl

```diff
@@ -45,6 +49,7 @@ steps:
                 default: true
             panel_of_normals_vcf:
                 default: null
+            tumor_sample_name: tumor_sample_name
```

Comment:
The `tumor_bam` input has `normal_bams` as its value.  Should this also be `normal_sample_name` to match?  (As an additional note, this tool now has both `tumor_sample_name` and `normal_sample_name` inputs but only one gets used.)